Formula/bareos-client.rb

```diff
@@ -21,29 +21,40 @@ class BareosClient < Formula
   depends_on "openssl@1.1"
   depends_on "readline"
 
-  conflicts_with "bacula-fd",
-    because: "both install a `bconsole` executable"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "acl"
+  end
+
+  conflicts_with "bacula-fd", because: "both install a `bconsole` executable"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args,
-                            "-Dworkingdir=#{var}/lib/bareos",
-                            "-Darchivedir=#{var}/bareos",
-                            "-Dconfdir=#{etc}/bareos",
-                            "-Dconfigtemplatedir=#{lib}/bareos/defaultconfigs",
-                            "-Dscriptdir=#{lib}/bareos/scripts",
-                            "-Dplugindir=#{lib}/bareos/plugins",
-                            "-Dfd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
-                            "-Dmon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
-                            "-Dbasename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dhostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dclient-only=ON"
-      system "make"
-      system "make", "install"
-    end
+    # Work around hardcoded paths to /usr/local Homebrew installation
+    inreplace "core/CMakeLists.txt", "/usr/local/", "#{HOMEBREW_PREFIX}/"
+    inreplace "core/cmake/FindReadline.cmake", "/usr/local/opt/readline/lib", Formula["readline"].opt_lib
```

Comment:
Hardcoded path to readline static library (`/usr/local/opt/readline/lib/libreadline.a`) in https://github.com/bareos/bareos/blob/Release/21.1.2/core/cmake/FindReadline.cmake#L32=

Could just remove the `Darwin` section and let CMake `find_library` properly detect the dylib.

Reply:
I'd suggest removing the Darwin section indeed